terraform-modules/global-ip-dns/test/test.tf

```diff
@@ -0,0 +1,14 @@
+provider "google" {
+  project = "broad-gotc-dev"
+}
+
+module "test_auth_proxy" {
+  source              = "../"
+
+  providers = { 
+    google = google
+    google.dns_provider = google
+  }
+  auth_proxy_dns_name = "test-auth-proxy"
+  auth_proxy_dns_zone = "gotc-dev"
```

Comment:
I believe test should be updated later on when we are all set with variables.

Reply:
yep